gitpod-io/gitpod
# 1427 - Add DNS and cert cleanup into the preview env cron

## Description
Added two functions to cleanup DNS records and certificates

## Related Issue(s)

Fixes #1427

## How to test
Create a new branch and then delete after all resources are created
Check the DNS and cert resources are correctly removed
